README.md

```diff
@@ -356,16 +356,58 @@ metadata:
   name: example-quayecosystem
 spec:
   quay:
-    routeHost: example-quayecosystem-quay-quay-enterprise.apps.openshift.example.com
+    hostname: example-quayecosystem-quay-quay-enterprise.apps.openshift.example.com
     imagePullSecretName: redhat-pull-secret
 ```
 
+### Methods for Exteral Access
+
+Support is available to access Quay through a number of OpenShift and Kubernetes mechanisms for ingress. When running on OpenShift, a [Route](https://docs.openshift.com/container-platform/4.2/networking/routes/route-configuration.html) is used while a [LoadBalancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) is used. 
```

Comment:
>is used while a LoadBalancer Service

This is a bit confusing to me. I assume it's supposed to say a `Route` is used on OpenShift and a `LoadBalancer Service` is used on Kubernetes. From reading it, it sounds like both a `Route` and `Loadbalancer Service` are used on Openshift. It may benefit from a little clarification.

Reply:
You can use either type